### ISSUE TITLE
Thumbnailing failed on Windows

### ISSUE THREAD

**Issue by Keith94** [TheAMM/mpv_thumbnail_script#1]
I get this error "thumbnailing failed, check console for details'. Not sure if I did something wrong.

Here's my log: https://hastebin.com/yofowemabe.md

And my `mpv_thumbnail_script.conf`:
```
# The thumbnail cache directory.
# On Windows this defaults to %TEMP%\mpv_thumbs_cache,
# and on other platforms to /tmp/mpv_thumbs_cache.
# The directory will be created automatically, but must be writeable!
# Use absolute paths, and take note that environment variables like %TEMP% are unsupported (despite the default)!
cache_directory=/tmp/my_mpv_thumbnails

# Whether to generate thumbnails automatically on video load, without a keypress
# Defaults to yes
autogenerate=yes

# Only automatically thumbnail videos shorter than this (in seconds)
# You will have to press T (or your own keybind) to enable the thumbnail previews
# Set to 0 to disable the check, ie. thumbnail videos no matter how long they are
# Defaults to 3600 (one hour)
autogenerate_max_duration=3600

# Use mpv to generate thumbnail even if ffmpeg is found in PATH
# It's better to use ffmpeg, but the choice is yours
# Defaults to no
prefer_mpv=yes

# Enable to disable the built-in keybind ("T") to add your own, see after the block
disable_keybinds=no

# The maximum dimensions of the thumbnails, in pixels
# Defaults to 200 and 200
thumbnail_width=200
thumbnail_height=200

# The thumbnail count target
# (This will result in a thumbnail every ~10 seconds for a 25 minute video)
thumbnail_count=150

# The above target count will be adjusted by the minimum and
# maximum time difference between thumbnails.
# The thumbnail_count will be used to calculate a target separation,
# and min/max_delta will be used to constrict it.

# In other words, thumbnails will be:
# - at least min_delta seconds apart (limiting the amount)
# - at most max_delta seconds apart (raising the amount if needed)
# Defaults to 5 and 90, values are seconds
min_delta=5
max_delta=90
# 120 seconds aka 2 minutes will add more thumbnails only when the video is over 5 hours long!
```

**Comment by TheAMM** [comment]
`cache_directory=/tmp/my_mpv_thumbnails` is not a Windows path. Don't blindly copy the example!

**Comment by Keith94** [comment]
My bad. Thanks for the reply.
